src/instrumenter.js

```diff
@@ -147,7 +147,13 @@ class Instrumenter {
   }
 
   _set (plugin, meta) {
-    meta = Object.assign({ config: {} }, meta)
+    const analytics = {}
+
+    if (typeof this._tracer._tracer._analytics === 'boolean') {
```

Comment:
We only want to enable web frameworks via the global `analytics` configuration.

Reply:
Yes, the plugins are configured to only use the default rate of 1 when it's a web framework.